### DISCUSSION TITLE
virustotal marked it as a trojan

### DISCUSSION THREAD

**Discussion by logancantspell** [TurboWarp/packager-extras#142] [General]
so i put it through virustotal and one of the vendors (zillya) said it was a Trojan.Pytr.Script.85
idk what that means but now idk if its safe
![emr eh](https://github.com/TurboWarp/packager-extras/assets/167826509/548eb463-0a92-4752-a2ac-1152bf0a598d)

**Comment by GarboMuffin** [comment]
Google autocomplete should give you a good idea about how trustworthy zillya is:

![image](https://github.com/TurboWarp/packager-extras/assets/33787854/e0bf026a-1e0f-48c5-ad18-3000e3d84c9e)

Another way to look at the 1/66 score is that 65 antivirus software vendors think it is safe.

**Comment by logancantspell** [comment]
thanks
